lib/pbench/cli/agent/commands/results/push.py

```diff
@@ -0,0 +1,61 @@
+import click
+
+from pbench.agent.base import BaseCommand
+from pbench.agent.results import CopyResultTb
+from pbench.cli.agent import pass_cli_context
+from pbench.cli.agent.options import common_options
+from pbench.common.logger import get_pbench_logger
+
+
+class ResultsPush(BaseCommand):
+    def __init__(self, context: click.Context):
+        super().__init__(context)
+
+    def execute(self) -> int:
+        logger = get_pbench_logger("pbench-agent", self.config)
+        crt = CopyResultTb(
+            self.context.controller, self.context.result_tb_name, self.config, logger
+        )
+        crt.copy_result_tb(self.context.token)
+        return 0
+
+
+@click.command()
+@common_options
+@click.option(
+    "--token",
+    required=True,
+    prompt=True,
+    envvar="PBENCH_ACCESS_TOKEN",
+    help="pbench server authentication token (will prompt if unspecified)",
+)
+@click.argument("controller")
+@click.argument(
+    "result_tb_name",
+    type=click.Path(exists=True, file_okay=True, dir_okay=False, readable=True),
+)
+@pass_cli_context
+def main(
+    context: click.Context, controller: str, result_tb_name: str, token: str,
```

Comment:
Until something changes, the type of `context` should (I think...) be `CliContext` (from `pbench.cli.agent`).

Reply:
It didn't look like you executed this suggestion.

If you're not going to change the type hint for `context`, then I suggest that you just remove it -- having it be wrong is likely to be worse than omitting it.

Reply:
(Line 11, too.)